VERSION

```diff
@@ -1 +1 @@
-3.13.0
+4.0.0
```

Comment:
Checking our release tooling (I'm sorry, it's been a while since I've looked at these) I think this remains 3.13.0 and gets bumped automatically https://github.com/aws/aws-sdk-ruby-release-tools/blob/2f3b0495701fca571d99896bd072a00257b710e7/release.rake#L47